app/components/contact/contact-item-modal.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class ContactContactItemModalComponent extends Component {
```

Comment:
Ik zou de componenten gerelateerd aan contact-items in een subfolder `contact-item` onder components zetten zodat ze gescheiden blijven van de `contact` components die voor bronnen gebruikt worden.